src/components/Layout/TableOfContents.jsx

```diff
@@ -60,58 +61,50 @@ const EntryTitle = styled.h6`
   line-height: 1.5;
   border-bottom: 1px solid transparent;
   text-decoration: none;
-`
+  `
 
 const ChapterListItem = styled.li`
   margin: 0;
-`
+  `
 
 const EntryListItem = styled.li`
   margin: 0;
-  a:hover {
+  a:hover
+  {
     border-bottom: 1px solid black;
   }
-`
+  `
 
 const ChapterTitle = styled.h5`
-  font-weight: ${({ level }) => {
-    switch (level % 3) {
-      case 1:
-        return '600'
-      case 2:
-        return '400'
-      default:
-        return '200'
-    }
-  }};
-  font-size: ${({ level }) => {
-    switch (level % 3) {
-      case 1:
-        return '2.4rem'
-      case 2:
-        return '2rem'
-      default:
-        return '2.8rem'
-    }
-  }};
-  color: ${({ level, theme }) => {
-    switch (level % 3) {
-      case 1:
-        return 'black'
-      case 2:
-        return 'blue'
-      default:
-        return theme.brand
-    }
-  }};
-  margin-bottom: ${({ level }) => {
-    switch (level % 3) {
-      case 1:
-        return '7px'
-      case 2:
-        return '5px'
-      default:
-        return '10px'
-    }
-  }};
+  font - weight
+: ${({level}) => {
+  switch (level % 3) {
+    case 1:
+      return '600'
+    case 2:
+      return '400'
+    default:
+      return '200'
+  }
+}};
+  font-size: ${({level}) => {
+  switch (level % 3) {
+    case 1:
+      return '2.2rem'
+    case 2:
+      return '2rem'
+    default:
+      return '2.8rem'
+  }
+}};
+  color: ${({level, theme}) => {
+  switch (level % 3) {
+    case 1:
+      return 'black'
```

Comment:
You see, one nearly misses this change, because one thinks that this PR only contains whitespace changes.

I would suggest you use a different color from `black` for on of the two levels to show people that they actually have the possibility to change this. This is why I used `blue`, which is ugly, I admit. Maybe `theme.warn` or `theme.primary` would be better? I do not know which colors are available.